call-center/web-client/src/hooks/useTelnyxRTC.ts

```diff
@@ -0,0 +1,121 @@
+import { useState, useRef, useEffect } from 'react';
+import { TelnyxRTC } from '@telnyx/webrtc';
+
+import IPartialWebRTCCall from '../interfaces/IPartialWebRTCCall';
+import { updateAgent } from '../services/agentsService';
+
+function useTelnyxRTC(
+  agentId: string | undefined,
+  token: string | undefined,
+  clearDialingDestination: any
+) {
```

Comment:
IMO it would be easier to understand the call signature if there was only one argument, you could set up an interface too:
```suggestion
interface IUseTelnyxRTCParams {
  agentId: string | undefined;
  token: string | undefined;
  clearDialingDestination: any;
}

function useTelnyxRTC({
  agentId,
  token,
  clearDialingDestination,
}: IUseTelnyxRTCParams) {
```

Reply:
Good idea, done!